libnavui-dropin/api/current.txt

```diff
@@ -0,0 +1,572 @@
+// Signature format: 3.0
+package com.mapbox.navigation.dropin {
+
+  public final class NavigationView extends android.widget.FrameLayout implements androidx.lifecycle.LifecycleOwner {
+    ctor public NavigationView(android.content.Context context, android.util.AttributeSet? attrs = null, String accessToken = attrs.navigationViewAccessToken(context));
+    ctor public NavigationView(android.content.Context context, android.util.AttributeSet? attrs = null);
+    ctor public NavigationView(android.content.Context context);
+    method public void customizeMapView(com.mapbox.maps.MapView? mapView);
+    method public void customizeViewBinders(kotlin.jvm.functions.Function1<? super com.mapbox.navigation.dropin.ViewBinderCustomization,kotlin.Unit> action);
+    method public void customizeViewOptions(kotlin.jvm.functions.Function1<? super com.mapbox.navigation.dropin.ViewOptionsCustomization,kotlin.Unit> action);
+    method public androidx.lifecycle.Lifecycle getLifecycle();
+  }
+
+  public final class NavigationViewContextKt {
```

Comment:
This should not be here 🤔

Reply:
maybe it a metalava issue with extension functions?

https://github.com/mapbox/mapbox-navigation-android/blob/9a655a77e5cf02ea6657792b7f4a30e477a71006/libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewContext.kt#L68-L73

Reply:
Fixed